server/index.js

```diff
@@ -28,7 +28,6 @@ if (process.env.NODE_ENV === 'test') {
  */
```

Comment:
I would keep all the comments in the server index.js. They are really helpful to keep track of which middleware is doing what. This file will be hard pretty hard to follow without them